movingpandas/trajectory_generalizer.py

```diff
@@ -200,3 +201,44 @@ def _generalize_traj(self, traj, tolerance):
         new_df = traj.df.iloc[keep_rows]
         new_traj = Trajectory(new_df, traj.id)
         return new_traj
+
+
+class TopDownTimeRatioGeneralizer(TrajectoryGeneralizer):
+    """
+    Generalizes using Top-Down Time Ratio algorithm proposed by Meratnia and De By (DOI: 10.1007/978-3-540-24741-8_44).
+
+    Points are projected on the line that connects the 
```

Comment:
This sentence appears incomplete

Reply:
Yeah I committed the wrong file. This should be fixed now. Sorry :)